frontend/src/helpers/GetScreenResolution.js

```diff
@@ -0,0 +1,8 @@
+function getScreenResolution() {
```

Comment:
In JS, components are named `ComponentName.jsx` and helpers and other JS files are named `helperName.js`.